app/routes.py

```diff
@@ -1,2 +1,205 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request
+from flask import request, Blueprint, make_response
+from flask import jsonify
+from datetime import datetime 
+from sqlalchemy import asc, desc
+import requests
+import json
+import os
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
 
+#This is is a new endpoint.
+@task_bp.route("/<task_id>", methods=["GET", "PUT", "DELETE"], strict_slashes=False)
+def get_single_task(task_id):
+    task = Task.query.get(task_id)
+
+    if task is None:
+        return jsonify(None), 404
```

Comment:
we can use a method that'll do both these things for us:

```suggestion
    task = Task.query.get_or_404(task_id)
```